addons/base/models.py

```diff
@@ -108,7 +108,7 @@ def nodes_authorized(self):
         model = self.config.node_settings
         if not model:
             return []
-        return [obj.owner for obj in model.objects.filter(user_settings=self, owner__is_deleted=False)]
+        return [obj for obj in model.objects.filter(user_settings=self, owner__is_deleted=False).select_related('owner')]
```

Comment:
[`select_related` just makes fetching the related objects not lazy.](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#select-related) Normally, django would only get the properties directly in the table of the model you're querying. When you try to access a FK or M2M it would do another sql query to get the relationship. `select_related` (fk) and `prefetch_related` (m2m) do the query or queries eagerly.